delivery_recipe/Delivery/README.md

```diff
@@ -1,2 +1,26 @@
 # Delivery
 
+Our delivery process is designed to handle any and all types of project. Rather than being tailored to a specific type of project our processes focus on what’s important in any project:
```

Comment:
The word 'project is used a lot in this page, especially this first sentence. As we're about delivering ongoing services, rather than short one-hit projects could we switch the wording round as in other pages in the cookbook?